src/Drush/Commands/OpenShiftDrushCommands.php

```diff
@@ -191,6 +194,7 @@ public function getDatabaseDump() : int {
         'drush',
         'sql:dump',
         '--structure-tables-key=common',
+        '--no-tablespaces',
```

Comment:
I'm trying to run this and I get this error on every instance: 

```
                                                 
  The "--no-tablespaces" option does not exist.  
                                                 
```

Should this be `--extra-dump="--no-tablespaces"`?